static/src/js/components/AdminRetrievals/__tests__/AdminRetrievalForm.test.jsx

```diff
@@ -1,29 +1,50 @@
-import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-import Form from 'react-bootstrap/Form'
+import { screen } from '@testing-library/react'
 
+import setupTest from '../../../../../../jestConfigs/setupTest'
 import { AdminRetrievalsForm } from '../AdminRetrievalsForm'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup() {
-  const props = {
-    onAdminViewRetrieval: jest.fn()
+const setup = setupTest({
+  Component: AdminRetrievalsForm,
+  defaultProps: {
+    onAdminViewRetrieval: jest.fn(),
+    onFetchAdminRetrievals: jest.fn()
   }
+})
+
+describe('AdminRetrievalsForm component', () => {
+  test('renders itself correctly', () => {
+    setup()
+  })
 
-  const enzymeWrapper = shallow(<AdminRetrievalsForm {...props} />)
+  test('onInputChange updates state', async () => {
+    const { user } = setup()
 
-  return {
-    enzymeWrapper,
-    props
-  }
-}
+    const retrievalInput = screen.getByPlaceholderText('Obfuscated Retrieval ID')
+    await user.type(retrievalInput, 'test-retrieval-id')
+    expect(retrievalInput).toHaveValue('test-retrieval-id')
+  })
 
-describe('AdminRetrievals component', () => {
-  test('renders itself correctly', () => {
-    const { enzymeWrapper } = setup()
+  test('onFormSubmit calls onFetchAdminRetrievals when userId provided', async () => {
+    const { props, user } = setup()
+
+    const userIdInput = screen.getByPlaceholderText('Enter User ID')
+    const goButtons = screen.getAllByText('Go')
+
+    await user.type(userIdInput, 'test-user-id')
+    await user.click(goButtons[1])
+
+    expect(props.onFetchAdminRetrievals).toHaveBeenCalledWith('test-user-id')
```

Comment:
`toHaveBeenCalledWith` should always be accompanied by `toHaveBeenCalledTimes`. It is important to know how many times you called a function to guard against unintended calls.

Sometimes you might have a function called multiple times on purpose, with different parameters. In that case you can use `toHaveBeenNthCalledWith(1, ...expected params)` to test that each call was made correctly